bundles/org.openhab.binding.miio/README.base.md

```diff
@@ -252,6 +252,14 @@ Additionally depending on the capabilities of your robot vacuum other channels m
 | Switch  | status#mop_forbidden_enable       | Mop Forbidden              |
 | Switch  | status#is_locating                | Robot is locating          |
 | Number  | actions#segment                   | Room Clean  (enter room #) |
+| Switch  | actions#collect_dust              | Start collect the dust at  |
+| Switch  | actions#clean_mop_star            | Start mop wash             |
+| Switch  | actions#clean_mop_stop            | Stop mop wash              |
+| Number  | status#mop_drying_time            | Mop Drying Time            |
+| Switch  | status#is_mop_drying              | Mop cleaning active        |
+| Number  | status#dock_state_id              | Dock Status id             |
+| String  | status#dock_state                 | Dock Status message        |
```

Comment:
Depending on decided casing:
```suggestion
| Number  | status#dock_state_id              | Dock status id             |
| String  | status#dock_state                 | Dock status message        |
```

Reply:
I changed them to lower case, even if some other descriptions using upper case.